### DISCUSSION TITLE
Request to Add zh_TW Language Support

### DISCUSSION THREAD

**Discussion by Angus-repo** [NightscoutFoundation/xDrip#3498] [Q&A] [answered]
Hello,

I would like to request the addition of the zh_TW language translation. However, I have been unable to get a response from the admin through Crowdin. Could you please help me add the zh_TW language support? I will upload the translated XML files once the language is added.

Thank you very much.

https://crowdin.com/project/xdrip

@jamorham

**Comment by Navid200** [comment]
Can we have this conversation in Crowdin?
I don't think we can add a language with no words translated yet.

**Reply by Angus-repo** [reply]
Hello,

I have already initiated this discussion on Crowdin:
https://crowdin.com/project/xdrip/discussions/18

**Reply by Navid200** [reply]
I have submitted a request in Crowdin to add the language.  I myself don't have the authority to add a language.

@jamorham

**Comment by jamorham** [comment]
Please excuse my ignorance on this subject but is this different to "Traditional Chinese" or "Traditional Chinese Hong Kong" because out of the box crowdin doesn't have an option for Taiwan?

**Reply by Angus-repo** [reply]
Thank you for your reply. These three languages are used by people in different regions, so the wording and grammar are different.
In Crowdin, you can find Chinese Traditional under Project → Settings → Languages. Simply check it and save the settings.

![Screenshot_20260125_112035~2](https://github.com/user-attachments/assets/89009b4d-cb5c-4feb-8171-39f1e4c152f2)
![Screenshot_20260125_112020~3](https://github.com/user-attachments/assets/9ba3b25a-4dcc-4203-b8f0-d152d4ef18ba)

**Comment by jamorham** [comment]
I've added traditional chinese to the list of supported languages.

**Comment by Angus-repo** [comment]
> I've added traditional chinese to the list of supported languages.

Thank you for enabling Traditional Chinese. I have already applied to join—please grant me translator and proofreader permissions. Thank you.

**Reply by Navid200** [reply]
We need at least two translators first before we can make them each other's proofreaders.  Isn't that the case?
If one person is the only translator as well as proofreader, couldn't it lead to unintended mistakes.

**Reply by Angus-repo** [reply]
This language may, in the short term, have only me translating and proofreading.

Other APS-related Traditional Chinese translations are also done by me alone (e.g., AAPS, AndroidAPSDocs, Trio, etc.).

Please grant me proofreader permissions first; I will carefully proofread this language.

**Reply by Navid200** [reply, marked as answer]
You are now a translator.  You should be able to go ahead and translate.
I approved the only request you had.

Please go ahead and translate and let me know what happens preferably in Crodwin discussions.

**Reply by Navid200** [reply]
Do you want to open a PR to add the language to the selector array?

**Reply by Angus-repo** [reply]
Yes, I will submit another PR to add the new language. Thank you.
